.github/workflows/export_database_views.yml

```diff
@@ -0,0 +1,77 @@
+name: Export Moped Views to SQL
+
+on:
+  push:
+    branches-ignore:
+      - "main"
+      - "production"
+
+jobs:
+  publish-views:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install SQLFluff
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-pip
+          pip3 install sqlfluff
+
+      - name: Install PostgreSQL client
+        run: sudo apt-get update && sudo apt-get install -y postgresql-client
+
+      - name: Set PostgreSQL environment variables
+        run: |
+          echo "PGHOST=localhost" >> $GITHUB_ENV
+          echo "PGPORT=5432" >> $GITHUB_ENV
+          echo "PGDATABASE=moped" >> $GITHUB_ENV
+          echo "PGUSER=moped" >> $GITHUB_ENV
+          echo "PGPASSWORD=moped" >> $GITHUB_ENV
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Install Hasura CLI
+        run: |
+          curl -L https://github.com/hasura/graphql-engine/raw/stable/cli/get.sh | bash
+
+      - name: "Spin-up the DB stack"
+        run: |
+          export BRANCH_NAME=${GITHUB_REF##*/}
+          echo "SHA: ${GITHUB_SHA}"
+          echo "ACTION/BRANCH_NAME: ${BRANCH_NAME}"
+          echo "GR: ${GITHUB_REF}"
+          echo "PWD: $(pwd)"
+          cd ./moped-database;
+          ./hasura-cluster setenv local;
+          ./hasura-cluster start;
+
+      - name: Generate CREATE VIEW statements
+        run: |
+          mkdir -p moped-database/views
+          psql -A -t -c "SELECT table_name FROM information_schema.views WHERE table_schema = 'public';" | \
+          grep -v -E '^geo[a-zA-Z]+y_columns$' | \
+          xargs -I {} sh -c "
+            VIEW_NAME={}
+            MOST_RECENT_MIGRATION=\$(grep -rl --include=up.sql -E 'CREATE (OR REPLACE )?VIEW (public.)?$VIEW_NAME' moped-database/migrations | sort -V | tail -n 1)
+            echo \"-- Most recent migration: \$MOST_RECENT_MIGRATION\" > moped-database/views/\$VIEW_NAME.sql
+            echo "" >> moped-database/views/\$VIEW_NAME.sql
+            psql -A -t -c \"SELECT 'CREATE VIEW ' || '\$VIEW_NAME' || ' AS ' || pg_get_viewdef('\$VIEW_NAME'::regclass, true);\" >> moped-database/views/\$VIEW_NAME.sql
+          "
+
+      - name: Run SQLFluff
+        run: |
+          sqlfluff format --config .sqlfluff moped-database/views/*.sql
+
+      - name: Set Git Config
+        run: |
+          git config --local user.email "moped@view.bot"
+          git config --local user.name "Moped View Bot"
```

Comment:
This is the identify of the bot when it creates commits. This data is low key, but I wanted to point it out in case y'all wanted to change the values.